go/mysql/constants.go

```diff
@@ -368,6 +368,7 @@ const (
 	ERServerShutdown = 1053
 
 	// not found
+	ERDbDropExists          = 1008
```

Comment:
Changes in this file were just me ordering the subgroups by error code as it makes it easier to walk through all of them as I did to see what ones should be in the `vreplication.unRecoverable()` list.